### DISCUSSION TITLE
Cannot install dependencies

### DISCUSSION THREAD

**Discussion by simerlec** [codesandbox/codesandbox-client#7508] [Q&A]
Hello 👋 

I have a sandbox created from a Vite template I like to use.

Template: https://codesandbox.io/p/sandbox/vite-react-typescript-starter-685son
Sandbox: https://codesandbox.io/p/sandbox/dreamy-https-gpjo0i

For some reason I cannot install additional dependencies in the sandbox using `npm install`.

![image](https://user-images.githubusercontent.com/5793380/225460026-8f43eed2-cc6a-4898-af8c-9bb1a8c88a63.png)

Can someone tell me what to do here?

thanks

**Comment by tdwaf** [comment]
You can install dependencies using yarn. So in your case, your install would look like `yarn add @preact/signals-react`
